README.md

```diff
@@ -0,0 +1,20 @@
+# CATcher Product Page
+
+# Setting up 
+This section guides you through the steps required to set up your computer for developing CATcher's Product Page.
```

Comment:
Hang on is it supposed to be for developing the product page 😂

Reply:
Like to work on this page 😅 I just followed the docs we had for our main repo 😆. What do you think is better?

Reply:
Oh icic, I thought it's for developing CATcher not the page oops 😅  Sounds good!!! 👍 👍